src/hooks/useHoldings.ts

```diff
@@ -21,18 +22,29 @@ const initalOtherToken = {
 
 const INITAL_DATA: HoldingsApi = {
   celo: {
-    custody: initalToken,
-    unfrozen: initalToken,
-    frozen: initalToken,
+    custody: initialToken,
+    unfrozen: initialToken,
+    frozen: initialToken,
   },
   otherAssets: [
```

Comment:
i know this wasn't strictly defined in the design but IMO the ordering seems quite random at the moment

- _either_ we should order by $ amount descending (i.e. largest holdings first, smallest last)
- _or_ at least group a bit more logically manually (i.e. USDT after sDAI feels off, intuitively would have expected from most to least used assets i.e. USDC => USDT => DAI => sDAI => euro stuff)